LICENSE.murmur2

```diff
@@ -0,0 +1,25 @@
+src/MurMurHash2.[ch]: git@github.com:abrandoned/murmur2.git
+
+
+MurMurHash2 Library
+//-----------------------------------------------------------------------------
+// MurmurHash2 was written by Austin Appleby, and is placed in the public
+// domain. The author hereby disclaims copyright to this source code.
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
```

Comment:
Does the following license really apply to public domain?
Did Austin Appleby specify this license for the code?

Reply:
yes AppleBy distributed his software under the MIT license.